docs/data_structures/libE_specs.rst

```diff
@@ -26,9 +26,15 @@ libEnsemble is primarily customized by setting options within a ``LibeSpecs`` cl
 
                 **comms** [str] = ``"mpi"``:
                     Manager/Worker communications mode: ``'mpi'``, ``'local'``, or ``'tcp'``.
+                    If ``nworkers`` is specified, then ``local`` comms will be used unless a
+                    parallel MPI environment is detected.
```

Comment:
Do we need the word "parallel" here?

Reply:
It checks there is more than one process. On some platforms you can srun into a bash session on a compute node and you are technically in an MPI environment. We don't want that to count.